test/src/helper/spawn.ts

```diff
@@ -240,7 +240,6 @@ export default class CodeChain {
                 `target/${useDebugBuild ? "debug" : "release"}/foundry`,
                 [
                     ...baseArgs,
-                    "--no-informer",
```

Comment:
This line is not related to this commit. Could you move it?

Reply:
I moved it to another commit.

Reply:
Great!